terraform/main.tf

```diff
@@ -1,12 +1,63 @@
-terraform {
-  required_providers {
-    kubernetes = {
-      source = "hashicorp/kubernetes"
-      version = "2.11.0"
-    }
-    google = {
-      source = "hashicorp/google"
-      version = "4.20.0"
-    }
+provider "google" {
+  project = var.project_id
+  region  = "europe-west1"
+}
+
+module "gke_auth" {
```

Comment:
My IntelliJ is complaining about the fact that these modules do not exist. I think the proper usage of terraform modules is linked to a directory? Do we care?

Reply:
It's not complaining in VS Code, and since it is working I dont really mind tbh